packages/pipeline/src/scripts/scrape-data.ts

```diff
@@ -0,0 +1,93 @@
+import { ExchangeEvents, ZeroEx } from '0x.js';
+import * as commandLineArgs from 'command-line-args';
+import * as queue from 'queue';
+import * as Web3 from 'web3';
+import { typeConverters } from '../utils.js';
+import { insertDataScripts } from './tables.js';
+
+const optionDefinitions = [{ name: 'from', alias: 'f', type: Number }, { name: 'to', alias: 't', type: Number }];
+const cli = commandLineArgs(optionDefinitions);
+
+const provider = new Web3.providers.HttpProvider('https://mainnet.infura.io/Rd44ElIrWp05rUczoKps');
```

Comment:
i think we should inject the API key via environment variables similar to `REDSHIFT_PASSWORD` so that other devs don't clone this and eat through our traffic allocation